packages/addons/addon-depends/multimedia-tools-depends/depends/libmediainfo/package.mk

```diff
@@ -3,6 +3,7 @@
 
 PKG_NAME="libmediainfo"
 PKG_VERSION="24.06"
+PKG_REV="1"
```

Comment:
unnecessary change

Reply:
That was too track any future changes since I could not find a model for creating .so files  I will remove it.